lib/main_containers.pm

```diff
@@ -316,6 +316,8 @@ sub load_helm_chart_tests {
         loadtest 'containers/charts/rmt';
     } elsif ($chart =~ m/private-registry/) {
         loadtest 'containers/charts/privateregistry';
+    } elsif ($chart =~ m/kiosk/) {
+        loadtest 'containers/charts/kiosk';
```

Comment:
Please rename to `kiosk-firefox` as there will be soon another `kiosk` container coming up :slightly_smiling_face:

Reply:
Yes